frontend/src/components/ReminderModalOptions/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChangeEvent } from "react";
+import * as Styled from "./style";
+
+interface Props {
+  isPadStart: boolean;
+  optionTexts: string[];
+  checkedText: string;
+  handleChangeText: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
리액트에서 제공해주는 ChangeEventHandler 타입이 있어요! 이걸로 바꿔도 될 듯싶네요! 저도 오늘 알앗어요 ㅎㅎ 

```suggestion
  handleChangeText: ChangeEventHandler<HTMLInputElement>
```

Reply:
좋은 타입을 제공해주고 있네요! 적용하도록 할게요~!😊

Reply:
관련 커밋 [103ae6f](https://github.com/woowacourse-teams/2022-pickpick/pull/409/commits/103ae6f4d84959b5f3b7ece996b10c03b0df28b0)